### ISSUE TITLE
Webpack compile error

### ISSUE THREAD

**Issue by ryancoughlin** [shukerullah/react-geocode#1]
I am getting the following message when my build tries to compile: 

```
./node_modules/react-geocode/index.js
Module parse failed: Unexpected token (48:18)
You may need an appropriate loader to handle this file type.
|    * @param {string} apiKey
|    */
|   setApiKey(apiKey: string): void {
|     API_KEY = apiKey;
|   },
```

This is triggered when I simply import the module not actually implementing it. 

Latest version of React and `create-react-app` - any ideas on what I could try to track this down?

**Comment by shukerullah** [comment]
Thanks for raising this issue. I believe this is related to [flow](https://github.com/facebook/flow). 

All my projects has flow so it works perfectly fine. I will remove flow type checking from this module in next update so it can work without any issue in all projects.

**Comment by shukerullah** [comment]
Fixed, please update this module to `0.0.2v`
